sdk_container/src/third_party/coreos-overlay/sec-policy/selinux-base/selinux-base-2.20221101-r4.ebuild

```diff
@@ -29,34 +28,27 @@ HOMEPAGE="https://wiki.gentoo.org/wiki/Project:SELinux"
 LICENSE="GPL-2"
 SLOT="0"
 
-# flatcar changes
 RDEPEND=">=sys-apps/policycoreutils-2.8
-	>=sys-apps/checkpolicy-2.8
-"
+	>=sys-apps/checkpolicy-2.8"
 DEPEND="${RDEPEND}"
-# flatcar: BDEPEND on python - normally pulled in through policycoreutils
-# but we made that dep conditional on USE=python
-BDEPEND="sys-devel/m4
-    ${PYTHON_DEPS}
-"
+BDEPEND="
+	${PYTHON_DEPS}
+	sys-devel/m4"
 
+S=${WORKDIR}/
 
-# flatcar changes
 PATCHES=(
 	"${FILESDIR}"/0001-policy-modules-kernel-all-more-actions-for-kernel.patch
-	"${FILESDIR}"/0001-policy-ms-MCS-restricts-relabelfrom.patch
 	"${FILESDIR}"/icmp-bind.patch
+	"${FILESDIR}"/files-relabel.patch
 )
```

Comment:
I think we probably could have this patches as user patches in `<coreos-overlay>/coreos/user-patches/sec-policy/selinux-base` directory.